src/firmware_new/boards/frankie_v1/BUILD

```diff
@@ -3,132 +3,417 @@ package(default_visibility = ["//visibility:public"])
+                "Drivers/STM32H7xx_HAL_Driver/Inc/stm32h7xx_hal_dma.h",
+                "Drivers/STM32H7xx_HAL_Driver/Inc/stm32h7xx_hal_dma_ex.h",
+                "Drivers/STM32H7xx_HAL_Driver/Inc/stm32h7xx_hal_eth.h",
+                "Drivers/STM32H7xx_HAL_Driver/Inc/stm32h7xx_hal_eth_ex.h",
+                "Drivers/STM32H7xx_HAL_Driver/Inc/stm32h7xx_hal_crc.h",
+                "Drivers/STM32H7xx_HAL_Driver/Inc/stm32h7xx_hal_crc_ex.h",
+                "Drivers/STM32H7xx_HAL_Driver/Inc/stm32h7xx_hal_exti.h",
+                "Drivers/STM32H7xx_HAL_Driver/Inc/stm32h7xx_hal_flash.h",
+                "Drivers/STM32H7xx_HAL_Driver/Inc/stm32h7xx_hal_flash_ex.h",
+                "Drivers/STM32H7xx_HAL_Driver/Inc/stm32h7xx_hal_gpio.h",
+                "Drivers/STM32H7xx_HAL_Driver/Inc/stm32h7xx_hal_gpio_ex.h",
+                "Drivers/STM32H7xx_HAL_Driver/Inc/stm32h7xx_hal_hsem.h",
+                "Drivers/STM32H7xx_HAL_Driver/Inc/stm32h7xx_hal_i2c.h",
+                "Drivers/STM32H7xx_HAL_Driver/Inc/stm32h7xx_hal_i2c_ex.h",
+                "Drivers/STM32H7xx_HAL_Driver/Inc/stm32h7xx_hal_mdma.h",
+                "Drivers/STM32H7xx_HAL_Driver/Inc/stm32h7xx_hal_pcd.h",
+                "Drivers/STM32H7xx_HAL_Driver/Inc/stm32h7xx_hal_pcd_ex.h",
+                "Drivers/STM32H7xx_HAL_Driver/Inc/stm32h7xx_hal_pwr.h",
+                "Drivers/STM32H7xx_HAL_Driver/Inc/stm32h7xx_hal_pwr_ex.h",
+                "Drivers/STM32H7xx_HAL_Driver/Inc/stm32h7xx_hal_rcc.h",
+                "Drivers/STM32H7xx_HAL_Driver/Inc/stm32h7xx_hal_rcc_ex.h",
+                "Drivers/STM32H7xx_HAL_Driver/Inc/stm32h7xx_hal_tim.h",
+                "Drivers/STM32H7xx_HAL_Driver/Inc/stm32h7xx_hal_tim_ex.h",
+                "Drivers/STM32H7xx_HAL_Driver/Inc/stm32h7xx_hal_uart.h",
+                "Drivers/STM32H7xx_HAL_Driver/Inc/stm32h7xx_hal_uart_ex.h",
+                "Drivers/STM32H7xx_HAL_Driver/Inc/stm32h7xx_ll_usb.h",
+                "stm32h7xx_hal_conf.h",
+                ],
+        defines = [
+                "STM32H743xx",
+                ],
+        includes = [
+                "./",
+                "Drivers/CMSIS/Device/ST/STM32H7xx/Include/",
+                "Drivers/CMSIS/Include/",
+                "Drivers/CMSIS/RTOS2/Include/",
+                "Drivers/CMSIS/RTOS2/Template/",
+                "Drivers/STM32H7xx_HAL_Driver/Inc",
+                "Middlewares/Third_Party/FreeRTOS/Source/include",
+                "Middlewares/Third_Party/FreeRTOS/Source/portable/GCC/ARM_CM4F/",
+                "Middlewares/Third_Party/LwIP/src/include",
+                "Middlewares/Third_Party/LwIP/system",
+                "Middlewares/Third_Party/LwIP/system/arch",
+                "Drivers/BSP/Components/lan8742",
+                ],
+        restricted_to = ["//cc_toolchain:stm32h7"],
+)
+
+cc_library(
+        name = "lwip_library",
+        srcs = [
+                "Middlewares/Third_Party/LwIP/system/OS/sys_arch.c",
```

Comment:
Can we glob these files at all?

Reply:
we could but there are files that we don't need as well in that folder, and the rest of the BUILD file was listing them out so I just added to it